src/isIsogram.test.js

```diff
@@ -6,4 +6,28 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
```

Comment:
not bad, but add 2-3 more tests with your own ideas

Reply:
Done